docs/client-certificate-setup.md

```diff
@@ -0,0 +1,67 @@
+## Client-Certificate based authentication
+
+If the certificate Authority and user certificates are already present the steps of creating the certificate authority and client certificates can be skipped.
+The following is an example of creating them.
+
+```bash
+cd /etc/ssl
+```
+### Create the Certificate Autority (CA)
+
+Firstly, generate the CA:      
+```openssl genrsa -aes256 -out ca.key 4096```          
+This asks to enter a passphrase.       
+Next, create the server-side certificate, that will be sent via the TLS server to the client.             
```

Comment:
Is that the CA cert or the server cert? If the latter, in most cases that will be an existing server certificate or from Let's Encrypt. Please take that into consideration and clarify/differentiate which steps are needed when using an existing server cert.